crates/ruma-client-api/src/dehydrated_device/delete_dehydrated_device.rs

```diff
@@ -0,0 +1,48 @@
+//! `DELETE /_matrix/client/*/dehydrated_device/`
```

Comment:
For context, unspecced as of now, but mentioned in https://github.com/matrix-org/matrix-spec-proposals/pull/3814/files#r1266444656.